seed_services_client/scheduler.py

```diff
@@ -3,74 +3,42 @@
 Client for Messaging Content Store HTTP services APIs.
 
 """
-import requests
-import json
+from .seed_services import SeedServicesApiClient
 
 
-class SchedulerApiClient(object):
-
+class SchedulerApiClient(SeedServicesApiClient):
     """
-    Client for Scheduler API.
-
-    :param str api_token:
+    Client for Scheduler Service.
 
-        An API Token.
+    :param str auth_token:
```

Comment:
Shouldn't this be in the same place as the `__init__` function? The `seed_services.py` file.

Reply:
The comments are already there if that's what you mean
https://github.com/praekeltfoundation/seed-services-client/blob/develop/seed_services_client/seed_services.py#L23
I can remove them from here if you think the redundancy is confusing?

Reply:
The problem is if we make any changes to the parameters of the `__init__` function we will have to update comments in multiple places.